src/CRSFforArduino.cpp

```diff
@@ -77,7 +64,7 @@ namespace sketchLayer
     bool CRSFforArduino::begin()
     {
 #if CRSF_RC_ENABLED > 0 || CRSF_TELEMETRY_ENABLED > 0
-        return _serialReceiver->begin();
+        return this->SerialReceiver::begin();
```

Comment:
Do I _really_ need to explicitly use `this` keyword for every call to any function from the Serial Receiver Layer here?

Reply:
Meh. I am leaving it here, because I have dragged this Pull Request on for long enough.  
If I went through and did _this,_ I am artificially creating unnecessary work for myself. IE The kind what serves no purpose other than to perpetuate procrastination and delaying higher priority tasks.

Throughout this project, I have continuously reminded myself to _not_ get bogged down with minutiae what makes about 🤏 _that_ much of a difference, in the grand scheme of things. So far, that has worked well, for me.